openrtb_ext/floors.go

```diff
@@ -0,0 +1,45 @@
+package openrtb_ext
+
+type PriceFloorSchema struct {
+	Fields    []string `json:"fields,omitempty"`
+	Delimiter string   `json:"delimiter,omitempty"`
+}
```

Comment:
Please be consistent with a line break between structs.

Reply:
Addressed